src/asyncOperationStateUtils.js

```diff
@@ -52,10 +55,44 @@ const updateAsyncOperationDescriptor = (state, descriptorOptions) => {
   };
 };
 
+const getLatestOperationByDescriptorId = (state, descriptorId) => {
+  return reduce(state.operations, (acc, operation) => {
+    if (operation.descriptorId === descriptorId) {
+      if (!acc) {
+        return operation;
+      }
+      if (acc.lastFetchStatusTime <= operation.lastFetchStatusTime) {
+        return operation;
+      }
+      return acc;
+    }
+  }, {});
+};
+
+const setInvalidatedOperationByKeyState = (state, asyncOperationKey, descriptorId) => {
```

Comment:
It's more like `createInvalidatedOperationByKeyState`, or even `createInvalidatedOperationState`

set seems like a confusing word since it's not actually setting the state.

Reply:
it's now `createInvalidatedOperationState`